js/app/modules/card/knowledgeDocument.js

```diff
@@ -237,7 +247,7 @@ const KnowledgeDocument = new Module.Class({
     _get_webview: function () {
         let webview = this._create_webview();
 
-        webview.renderer.enable_scroll_manager = this.show_toc;
+        webview.renderer.enable_scroll_manager = true;
```

Comment:
hmm, at least in the way this is implemented it depends on `scrollTo` which is defined in the manager, which is disabled if the TOC is not visible, so it simply won't work.

I thought the actual scroll was handled [here](https://github.com/endlessm/eos-knowledge-lib/blob/master/data/templates/js/scroll-manager.js#L39) in the manager.